blackbox/docs/sql/system.txt

```diff
@@ -1111,41 +1112,36 @@ SQL statement needed to restore the table, like so::
 
 Here, ``table_ident`` is the name of the table you want to recreate.
 
-Copy the output of this command to somewhere safe. You will need it to recreate
-the table.
+Copy the output of this command, replace the ``table_ident`` with
+``table_ident_new`` and execute it to create a new table identical to the one
+you want to recreate.
 
-Next, use :ref:`copy_to` to export the table data to a file.
+Make sure you stop inserting data to the original ``table_ident``.
```

Comment:
shall we suggest blocking writes altogether? or at least point to the configuration that would enable them to do it?